### DISCUSSION TITLE
Characteristics information for each Bible version

### DISCUSSION THREAD

**Discussion by mmartin9684-sil** [BibleNLP/bible-graph#5] [Ideas]
Here are a few of the additional characteristics that would be helpful to have available for each Bible version:

- Language related characteristics, such as language family / hierarchy, language country, typology (e.g., SOV)
- Publication characteristics, such as publication date, copyright / licensing, description, some indication of scope and/or list of books, script,

**Comment by DCNemesis** [comment]
Generally agree on these. On the first one, I think we'd want a branch that captures language family hierarchy, and one that captures gateway languages (I was going to make a comment to that effect and I saw this one). On language country and typology ... possibly useful. I guess there's a question of how far we want to go with this stuff that is largely captured in Ethnologue. As far as making wise use of institutional resources, the in depth language knowledge is probably best kept in Ethnologue, and Ethnologue should be fitted with a useful API to retrieve that information, rather than duplicating (and probably falling behind) the source of this information.

On publication characteristics, I think there is some stuff in the schema that captures some of this. Like the list of books, script, etc. are captured by the graph, not in any one node. The API should be constructed to make retrieving some of those things easier, but it doesn't necessarily all need to be summarized in graph nodes.
